pam_eye.c

```diff
@@ -7,45 +8,87 @@
 #include <security/pam_modules.h>
 
 #define USERAGENT "pam_eye/1.0"
-#define TIMEOUT   10L
+#define DEFAULT_TIMEOUT_MS 200
```

Comment:
But if timeout is 2seconds or more, then when pam_eye has problems with connection, user will see "something went wrong" and will report about this.